{"name": "test_login_data_driven[TC005]", "status": "passed", "description": "\nData-Driven Login Test using Excel\nCovers:\n- Valid logins (standard_user, problem_user, etc.)\n- Invalid logins (locked_out_user, empty fields, wrong creds)\n", "attachments": [{"name": "log", "source": "6ef85c7e-5364-4b3c-84a5-fd33e277ed5a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_case", "value": "{'TestCaseID': 'TC005', 'Username': 'invalid_user', 'Password': 'wrong_pass', 'ExpectedResult': 'Error'}"}], "start": 1761114533554, "stop": 1761114537283, "uuid": "b22ba0d8-6b31-4d20-94e7-3848cbfb4e8b", "historyId": "c063b98bedbe1fb0e423a47608b9d2ad", "testCaseId": "e61e91c8e7cf75b1f54487af25e7b0f4", "fullName": "tests.test_login_data_driven#test_login_data_driven", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_data_driven"}, {"name": "host", "value": "ASCENDI-8RP1I77"}, {"name": "thread", "value": "6920-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_data_driven"}], "titlePath": ["tests", "test_login_data_driven.py"]}